README.md

```diff
@@ -40,32 +40,32 @@
  On this page display the names of files & folders (as determined by an API call) for the selected usernames’ repository.
  
  ## Task 7.
-  Stretch exercise.
+  //Stretch exercise.
```

Comment:
What does "Stretch exercise" mean?

Reply:
parts i copied from James original post but you are right this is not needed